yarn.lock

```diff
@@ -1874,6 +1893,26 @@
     superstruct "^0.14.2"
     tweetnacl "^1.0.0"
 
+"@solana/web3.js@^1.34.0":
```

Comment:
@ThomasProust so we add yet another version of `@solana/web3.js` in the project :(

As much as possible, need to either update all other dependents of `@solana/web3.js` so it's deduped, or, downgrade it to the lowest common version that can be deduped, between the dependents.

Reply:
We need to update the web3 version to match Raydium's, otherwise it doesn't work. I guess it's best to update all other dependencies